test/OrchardCore.Tests/Roles/DefaultSystemRoleProviderTests.cs

```diff
@@ -5,16 +5,15 @@ namespace OrchardCore.Roles.Tests;
 public class DefaultSystemRoleProviderTests
 {
     [Fact]
-    public void GetSystemRoles_Have_Administrator_Authenticated_Anonymous()
+    public void GetSystemRoles_WhenCalledByDefault_ContainsAdministratorAuthenticatedAndAnonymousRoles()
```

Comment:
I dislike "WehnCalled" because of course you will cal the method

Reply:
The standard naming convention, MethodName_ActionName_Result

Reply:
Which standard do you mean? Second these change doesn't related to the PR title

Reply:
https://learn.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices

I don't want to submit many PR to do one thing which is related to the latest commit.

Reply:
There are not many PRs, just two :) It would be better that a PR fixes a specific issue, is easy to review, and will be merged quickly